man/setDT.Rd

```diff
@@ -58,6 +59,14 @@ setDT(X, key="a")[]
 X = list(a=1:5, a=6:10)
 setDT(X, check.names=TRUE)[]
 
+# FAQ: Dealing with data.table loaded from RDS or RData files
+# Example to demonstrate the usage of setDT after loading from RDS
+if (file.exists("my_data.rds")) {
```

Comment:
Examples should be executable therefore writing before and loading it afterwards makes more sense. Check e.g. `help(readRDS)`